src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPool.cs

```diff
@@ -668,6 +669,79 @@ internal void Clear()
                 _cleanupWait,
                 _cleanupWait);
 
+        private static readonly string[] AzureSqlServerEndpoints = {SR.GetString(SR.AZURESQL_GenericEndpoint),
+                                                                      SR.GetString(SR.AZURESQL_GermanEndpoint),
+                                                                      SR.GetString(SR.AZURESQL_UsGovEndpoint),
+                                                                      SR.GetString(SR.AZURESQL_ChinaEndpoint) };
+        private static bool IsAzureSqlServerEndpoint(string dataSource)
```

Comment:
This method exists in AdapterUtil.SqlClient.cs It would be better to reuse it

Reply:
done.